Sources/MapboxCoreNavigation/DistanceFormatter.swift

```diff
@@ -1,26 +1,26 @@
 import CoreLocation
 import MapboxDirections
 
-struct RoundingTable {
-    struct Threshold {
-        let maximumDistance: Measurement<UnitLength>
-        let roundingIncrement: Double
-        let maximumFractionDigits: Int
+public struct RoundingTable {
+    public struct Threshold {
+        public let maximumDistance: Measurement<UnitLength>
+        public let roundingIncrement: Double
+        public let maximumFractionDigits: Int
         
-        func measurement(of distance: CLLocationDistance) -> Measurement<UnitLength> {
+        public func measurement(of distance: CLLocationDistance) -> Measurement<UnitLength> {
             var measurement = Measurement(value: distance, unit: .meters).converted(to: maximumDistance.unit)
             measurement.value.round(roundingIncrement: roundingIncrement)
             measurement.value.round(precision: pow(10, Double(maximumFractionDigits)))
             return measurement
         }
     }
     
-    let thresholds: [Threshold]
+    public let thresholds: [Threshold]
```

Comment:
In order for the developer to have any control over the rounding behavior, the `metric`, `uk`, and `us` static properties below need to be public as well.

Reply:
Couldn't they create extensions like so?

```
extension Measurement where UnitType == UnitLength {
    public func localized() -> Measurement<UnitLength> {
        let threshold: RoundingTable = .custom
        return threshold.threshold(for: distance).measurement(of: distance)
    }
}

extension RoundingTable {
    static var custom: RoundingTable = RoundingTable(thresholds: [
         // thresholds go here
    ])
}
```

Reply:
I think the idea is for developers to be able to change the behavior of existing UI elements, right? If so, they need to hook into whatever is already being called. It isn’t possible to reimplement a method in quite that way; you’d need to swizzle the method, something we wouldn’t want to encourage.

Reply:
Ah I see. Thank you for clarifying.